roles/pulp_webserver/tasks/main.yml

```diff
@@ -13,6 +13,26 @@
   tags:
     - always
 
+- import_tasks: generate_tls_certificates.yml
+  when:
+    - pulp_webserver_ssl_cert is undefined
+    - pulp_webserver_ssl_key is undefined
+
+- import_tasks: import_certificates.yml
+  when:
+    - pulp_webserver_ssl_cert is defined
+    - pulp_webserver_ssl_key is defined
```

Comment:
The "when" logic has a loophole. What if one of the variables is defined and the other is not?
Can we capture those cases with assert?